part-spec/sensor.json

```diff
@@ -0,0 +1,240 @@
+                    "description": "current consumption of a device",
+                    "type": "array",
+                    "items": {
+                      "$ref": "definitions.json#/currentConsumption"
+                    }
+                },
+                "package": {
+                    "description": "package size of a device",
+                    "$ref": "definitions.json#/package"
+                },
+                "pins": {
+                    "description": "array of pin objects with associated properties",
+                    "type": "array",
+                    "items": {
+                        "$ref": "definitions.json#/pinSpec"
+                    }
+                }
+            }
+        },
+        {
+            "title": "gyro",
+            "type": "object",
+            "required": [
+                "componentID"
+            ],
+            "properties": {
+                "componentID": {
+                    "description": "common component identifying information, such as mpn",
+                    "$ref": "definitions.json#/componentID"
+                },
+                "gyroRanges": {
+                    "description": "range of angular speed that gyro can measure",
+                    "examples": [
+                        "125",
+                        "250",
+                        "500"
+                    ],
+                    "comment": "units of degrees-per-second (dps)",
+                    "type": "number"
+                },
+                "gyroSensitivity": {
+                    "description": "smallest change in angular speed gyro is able to capture (typical). Corresponds to levels in gyro ranges",
+                    "examples": [
+                        "16",
+                        "32",
+                        "65"
+                    ],
+                    "comment": "units of LSB/dps",
+                    "type": "number",
+                    "$ref": "definitions.json#/conditionalProperty"
+                },
+                "gyroSensitivityOverTemperature": {
+                    "description": "gyroscope sensitivity change over temperature",
+                    "comment": "units of percentage per degree celsius",
+                    "$ref": "definitions.json#/value"
+                },
+                "axis": {
+                    "description": "number of axes of measurement",
+                    "type": "number"
+                },
+                "zeroRateOffset": {
+                    "description": "output of the gyroscope when no angular velocity is applied",
+                    "comment": "units of degrees-per-second",
+                    "$ref": "definitions.json#/value"
+                },
+                "zeroRateOffsetOverTemperature": {
+                    "description": "gyro zero rate offset change over temperature",
+                    "comment": "units of degrees-per-second per degree celsius (dps/Celcius)",
```

Comment:
I'm probably confused: `degrees-per-second per degree celsius` sounds like `celsius-per-second per celsius` which would be `C/s / C` => `1/s` => `Hz`.

Unless this is somehow very obvious to folks familiar with gyroscopes, can we update this comment to be more clear? (I don't really have a great suggestion because I think I just don't understand this :] )

Reply:
degrees (angle) vs degrees ( temperature). But yes, it can be confusing.

Reply:
oh I see, thanks!